OpenOversight/tests/conftest.py

```diff
@@ -81,17 +81,40 @@ def pick_salary():
     return random.randint(100, 100000000) / 100
 
 
+def pick_date(seed: bytes = None, start_year=2000, end_year=2020):
+    # source: https://stackoverflow.com/questions/40351791/how-to-hash-strings-into-a-float-in-01
+    # Wanted to deterministically create a date from a seed string (e.g. the hash or uuid on an officer object)
+    from struct import unpack
+    from hashlib import sha256
+
+    def bytes_to_float(b):
+        return float(unpack('L', sha256(b).digest()[:8])[0]) / 2 ** 64
+
+    if seed is None:
+        seed = str(uuid.uuid4()).encode('utf-8')
+
+    return datetime.datetime(start_year, 1, 1, 00, 00, 00) \
+        + datetime.timedelta(days=365 * (end_year - start_year) * bytes_to_float(seed))
+
```

Comment:
Nice approach. If we want to go with this I think we should put the `bytes_to_float` function in a utils-file are something so that we can use it everywhere, and I suggest rewriting the function as 
```
    def bytes_to_float(seed = None):
        if seed is None:
            seed = uuid.uuid4()
        return float(unpack('L', sha256(str(seed).encode('utf-8')).digest()[:8])[0]) / 2 ** 64
```
that way any object that has a `str` function can be used as seed.

if we want the result to be deterministic we have to make sure that the input is deterministic too. In this case it seems that the last name is used, which is unfortunately randomly chosen.